src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -101,6 +112,12 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// <remarks>This property is used only for unit testing.</remarks>
         internal IConfigurationClientManager ClientManager { get; set; }
 
+        /// <summary>
+        /// An optional class used to process pageable results from Azure App Configuration.
+        /// </summary>
+        /// <remarks>This property is only set outside of this class if it's used for unit testing.</remarks>
+        internal ConfigurationSettingPageableManager PageableManager { get; set; } = new ConfigurationSettingPageableManager();
```

Comment:
It's not managing anything, it's for custom iteration. I recommend `IConfigurationSettingsPageIterator`

Reply:
Or just `IPageableConfigurationSettings` since it's to mock `AsyncPageable<ConfigurationSetting>`